operator/cmd/mesh/manifest-generate_test.go

```diff
@@ -193,35 +193,47 @@ func TestMain(m *testing.M) {
 func TestManifestGenerateComponentHubTag(t *testing.T) {
 	g := NewWithT(t)
 
-	objs, err := runManifestCommands("component_hub_tag", "", liveCharts, []string{"templates/deployment.yaml"})
+	objs, err := runManifestCommands("component_hub_tag", "", liveCharts, []string{"templates/deployment.yaml", "templates/daemonset.yaml"})
```

Comment:
FWIW in looking at this now I realize there are ways in which this will fail that `helm template` will not fail - which is another good reason for

- Not doing any munging in `istioctl` at all, and doing it in Helm templates (like https://github.com/istio/istio/pull/51517)
- not carrying two templating tools that produce different results for the same inputs.